docs/data/charts/axis/axis.md

```diff
@@ -191,6 +191,33 @@ At the bottom, you can see one tick for the beginning and the middle of the day
 
 {{"demo": "TickLabelPosition.js"}}
 
+## Position
+
+The axis position can be customized with the `position` property of the axis configuration.
+Its value can be:
+
+- `'top'` or `'bottom'` for the x-axis.
+- `'left'` or `'right'` for the y-axis.
+- `'none'` to hide the axis.
+
+{{"demo": "ModifyAxisPosition.js"}}
+
+### Hiding axis
+
+To hide an axis, set its `position` to `'none'`.
+The axis is still computed and used for the scaling.
+
+{{"demo": "HidingAxis.js"}}
+
+### Multiple axes on the same side
+
+You can display multiple axes on the same side.
+If two or more axes share the same `position`, they are displayed in the order they are defined from closest to the chart to farthest.
+
+To avoid overlapping, you can use the `height` prop for `xAxis` and `width` for `yAxis` to increase the space between the axes.
```

Comment:
@alexfauquette I mention the axis size in this part, do you think it should have a separate example? Or do you think a visual explanation like the image below should be added?

![Frame 2 (2)](https://github.com/user-attachments/assets/7431f9f5-6156-4467-9bc6-23cdc53f7264)

Reply:
No, that one is nice. I was thinking about [this section](https://next.mui.com/x/react-charts/styling/#placement) that should be updated.

For example, the following sentence is now wrong

> You might want to modify those values to leave more space for your axis ticks or reduce them to provide more space for the data.